postgres-express-setup-tutorial.md

```diff
@@ -19,70 +19,84 @@ summary = "The article is a checklist for myself, but for anyone else setting up
+The article started as a checklist I put together for myself, which I've decided to share with everyone who needs help setting up PostgreSQL on MacOS, to use SQL database in a Express.js application. If you're just setting up Postgres on MacOS, you can leave out the part about Express. If you attempt to run Postgres on Windows, you should be able to substitute most of the tools here for your machine. You might be wondering why there's a need for another setup guide for Postgres in the first place, since there are a few across the web. I found many tutorials and guides on how to set it up, but found most of them to be unclear and outdated, based on old Postgres versions. The checklist is not comprehensive, though, as it pertains to my own way of doing things, so any feedback on simpler or more universal means is always welcome.
 
 {{% chapter_header "Postgres Setup on MacOS" "postgres-setup-macos" %}}
 
-My personal recommendation is using {{% a_blank "Homebrew" "https://brew.sh/" %}} for installing and managing your applications on MacOS. Over here you can [find some help on how to setup Homebrew](https://www.robinwieruch.de/developer-setup/). Afterward, you can update all your Homebrew dependencies and install PostgreSQL on the command line:
+I recommend {{% a_blank "Homebrew" "https://brew.sh/" %}} for installing and managing applications on MacOS. It is installed using the following command in the MacOS terminal:
+
+{{< highlight javascript >}}
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+{{< /highlight >}}
+
+The terminal runs through a series of installation operations, and will probably create folders in your local machine to accommodate Homebrews storage requirements. You can find more detailed instructions [here](https://www.robinwieruch.de/developer-setup/). After it's installed, update  the Homebrew dependencies and install PostgreSQL on the command line:
 
 {{< highlight javascript >}}
 brew update
 brew install postgresql
 {{< /highlight >}}
 
-Then you can check your PostgreSQL version on the command line:
+Next, check your PostgreSQL version:
 
 {{< highlight javascript >}}
 postgres --version
-postgres (PostgreSQL) 10.3
+postgres (PostgreSQL) 11.1
 {{< /highlight >}}
 
-As you can see, it is all about PostgreSQL 10 in the following checklist. But I am looking forward to update this checklist once Postgres receives another update.
+The version at the time of this publication is PostgreSQL 11.1.  I recommed using the latest version of libraries and software whenever possible to avoid compatibility issues with client-side applications.
```

Comment:
"The version at the time of this publication is PostgreSQL 11.1." -> "That's the version at the time of this publication." 

Reason: If the version changes, I only have to adjust it at one place and don't miss the other place.